CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.0.0-preview.5 (Unreleased)
+
+
```

Comment:
Having the extra new line will fail the markdown linter if (or once) we enable it in CI. Maybe we should fix this to only have one line.